K8s-deployment/Charts/auditing-server/README.md

```diff
@@ -2,6 +2,8 @@
 
 Helm Chart for IUDX auditing-server Server Deployment
 
+[![Kubescape Status](https://img.shields.io/jenkins/build?jobUrl=https%3A%2F%2Fjenkins.iudx.io%2Fview%2FKubescape%2Fjob%2Fkubescape-auditing%2F&label=Kubescape)](https://jenkins.iudx.io/job/kubescape-auditing/lastBuild/)
```

Comment:
We need the badges to be linked to the Kubescape reports of the last build, not the build itself.